encryption/main.cpp

```diff
@@ -0,0 +1,17 @@
+#include <iostream>
+#include "caesar.h"
+
+int main() {
+    std::cout << "Enter a message:" << std::endl;
+    std::string message;
```

Comment:
Don't forget add includes for all instruments, that you use, here, for example, no include for string, please check all you changes about this point.

Reply:
Not changed